### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by OlesiaSub** [objectionary/eo#1238]
Here is console log after `eoc dataize app` command. (I did `eoc clean` right before it)
```
Exception in thread "Thread-2" java.lang.NoClassDefFoundError: org/glassfish/json/JsonArrayBuilderImpl
	at org.glassfish.json.JsonProviderImpl.createArrayBuilder(JsonProviderImpl.java:203)
	at javax.json.Json.createArrayBuilder(Json.java:262)
	at com.yegor256.tojos.MnJson.write(MnJson.java:128)
	at com.yegor256.tojos.MnPostponed.lambda$start$1(MnPostponed.java:143)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.ClassNotFoundException: org.glassfish.json.JsonArrayBuilderImpl
	at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
	at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
	... 5 more
EO objects registered in /home/olesya/example/.eoc/eo-foreign.json
EO program assembled in /home/olesya/example/.eoc
Java sources generated in /home/olesya/example/.eoc/generated-sources
Java .class files compiled into /home/olesya/example/.eoc
Executable JAR created at /home/olesya/example/.eoc/eoc.jar
Error: Could not find or load main class org.eolang.Main
Caused by: java.lang.ClassNotFoundException: org.eolang.Main
```

Code in the app.eo is primitive:
```
[] > app
  QQ.io.stdout > @
    "hello"

```

**Comment by MikhailLipanin** [comment]
@yegor @OlesiaSub I fixed it by updating `eoc` version, 
via:
```
npm install -g eolang
```

**Comment by OlesiaSub** [comment]
@MikhailLipanin it helped, thanks!

@yegor256 is it still considered a bug or I can close the issue?

**Comment by yegor256** [comment]
@OlesiaSub it was a bug, but I fixed it in #1240
